entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -296,7 +296,8 @@ class EntityHandler(
             .then(body)
             .map {
                 val contexts = checkAndGetContext(httpHeaders, it)
-                entityService.updateEntityAttribute(entityId.toUri(), attrId, it, contexts)
+                val expandedPayload = expandJsonLdFragment(it, contexts) as Map<String, List<Any>>
```

Comment:
It would be clearer and more efficient to directly expand a `mapOf(attrId to it)` (attrId being later expanded in `partialUpdateEntityAttribute(...)`)